src/main/java/org/skife/config/ConfigurationObjectFactory.java

```diff
@@ -188,6 +194,7 @@ private void buildSimple(List<Callback> callbacks, Method method, Config annotat
         //   ignore the passed in value (which will be null)
         // - if all else fails, throw an exception.
```

Comment:
Could you expand this explanatory comment to make accurate for this feature? The code is hairy enough that I would like to the keep the explanation in sync.